rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -118,6 +118,31 @@
           </div>
         </div>
         <hr class="spacer" />
+        <div class="deca">
+          <h3>Protocol</h3>
+          <div class="lift:administration.PropertiesManagement.networkProtocolSection" id="networkProtocolForm">
+            <form class="lift:form.ajax">
+              <div class="wbBaseField">
+                    <label for="reportProtocol" class="threeCol textight" style="font-weight:bold;width: 50%;">
+                      <span >Use UDP in place of TCP for syslog communication between node and servers:
+                        <img src="/images/icInfo.png" style="padding-left:15px; margin:0; float:none"  tooltipid="reportProtocolTooltip" title="" class="tooltipable" />
+                      </span>
+                     <div class="tooltipContent" id="reportProtocolTooltip">
+                     Rudder uses syslog to transfer report messages from nodes to relay and
+             servers. By default, syslog uses TCP, which allows to minimize the loss in reports.
```

Comment:
By default, Rudder configure syslog to use UDP so that the syslog never stall an a non responsive connection, which could also stale other services on the node which use syslog. On the other hand, using TCP allows to minimize the loss in reports and the resulting compliance errors.